rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json

```diff
@@ -210,130 +209,171 @@
         ]
       }
     },
-    "default_contact": {
-      "title": "Default contact details",
-      "type": "object",
-      "propertiesOrder": [
-        "contact_person",
-        "street",
-        "postal_code",
-        "city",
-        "country",
-        "phone",
-        "email"
-      ],
-      "required": [
-        "city"
-      ],
-      "properties": {
-        "contact_person": {
-          "title": "Contact person",
-          "description": "Vendor contact person.",
-          "type": "string",
-          "minLength": 4
-        },
-        "street": {
-          "title": "Street",
-          "description": "Street and number of the address, separated by a coma.",
-          "type": "string",
-          "minLength": 4,
-          "form": {
-            "type": "textarea",
-            "templateOptions": {
-              "rows": 2
+    "contacts": {
+      "title": "Contacts",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "title": "Contact",
+        "type": "object",
+        "propertiesOrder": [
+          "type",
+          "contact_person",
+          "street",
+          "postal_code",
+          "city",
+          "country",
+          "phone",
+          "email"
+        ],
+        "required": [
+          "type",
+          "city"
+        ],
+        "properties": {
+          "type": {
+            "title": "Contact type",
+            "type": "string",
+            "default": "default",
+            "enum": [
+              "default",
+              "order",
+              "serial"
+            ],
+            "form": {
+              "type": "selectWithSort",
+              "options": [
+                {
+                  "value": "default",
+                  "label": "Default contact "
+                },
+                {
+                  "value": "order",
+                  "label": "Order contact"
+                },
+                {
+                  "value": "serial",
+                  "label": "Serial contact"
+                }
+              ],
+              "templateOptions": {
+                "itemCssClass": "col-lg-12",
+                "selectWithSortOptions": {
+                  "order": "label"
+                }
+              }
+            }
+          },
+          "contact_person": {
+            "title": "Contact person",
+            "description": "Vendor contact person.",
+            "type": "string",
+            "minLength": 4,
+            "form": {
+              "templateOptions": {
+                "itemCssClass": "col-lg-12",
+                "addonLeft": {
+                  "class": "fa fa-user"
+                }
+              }
+            }
+          },
+          "street": {
+            "title": "Street",
+            "description": "Street and number of the address, separated by a coma.",
+            "type": "string",
+            "minLength": 4,
+            "form": {
+              "type": "textarea",
```

Comment:
because in the street zone, we also enter the number with potential additional data (bte, level, ...)
Sometimes, users need multiple rows for this field --> textarea